src/pages/driver/luggage-policies/index.tsx

```
@@ -140,76 +139,54 @@ const LuggagePolicyRow = ({
+        ? Math.round(policies.reduce((acc, p) => acc + (p.excessFeePerKg || 0), 0) / policies.length * 100) / 100
+        : 0
   };
 
-  const handleDeletePolicy = (policyId: string) => {
+  const handleDeletePolicy = async (policyId: string) => {
     if (window.confirm('Are you sure you want to delete this luggage policy? This action cannot be undone.')) {
-      deletePolicyMutation.mutate(policyId);
+      try {
+        await deletePolicyMutation({ policyId });
+        toast.success('Luggage policy deleted successfully');
+      } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)